tools/generate-workspace/src/cli.ts

```diff
@@ -1,36 +1,18 @@
-import { findWorkspacePackages } from "@pnpm/workspace.find-packages";
-import { findWorkspaceDir } from "@pnpm/find-workspace-dir";
-import { writeFileSync } from "fs";
-import { join } from "path";
+import {findWorkspacePackages} from "@pnpm/workspace.find-packages";
+import {findWorkspaceDir} from "@pnpm/find-workspace-dir";
+import {writeFileSync} from "fs";
+import {join} from "path";
 
 // Define ignore list and name mapping
-const ignoreList = [
-  "@wingconsole/design-system",
-  "@wingconsole/error-message",
-  "@wingconsole/eslint-plugin",
-  "@wingconsole/tsconfig",
-  "@wingconsole/use-loading",
-  "@wingconsole/use-persistent-state",
-  "@winglang/platform-awscdk",
-  "@winglang/tree-sitter-wing",
-  "@winglang/wingc",
-  "@winglang/wingii",
-  "@winglibs/testfixture",
-  "bump-pack",
-  "construct-library",
-  "generate-workspace",
-  "hangar",
-  "jsii-fixture",
-  "ts-fixture",
-  "ts4w",
-];
+const ignoreList = ["@wingconsole/design-system", "@wingconsole/error-message", "@wingconsole/eslint-plugin", "@wingconsole/tsconfig", "@wingconsole/use-loading", "@wingconsole/use-persistent-state", "@winglang/platform-awscdk", "@winglang/compatibility-spy", "@winglang/tree-sitter-wing", "@winglang/wingc", "@winglang/wingii", "@winglibs/testfixture", "bump-pack", "construct-library", "generate-workspace", "hangar", "jsii-fixture", "ts-fixture", "ts4w"];
```

Comment:
Was this collapse intentional?

Reply:
prettier did it, the file wasn't changed since we added it probably